OmiseSDK/Compatibility/OmiseSource.swift

```diff
@@ -430,12 +426,6 @@ extension PaymentInformation {
             switch value.type {
             case .internetBankingBAY:
                 bank = .bay
-            case .internetBankingKTB:
-                bank = .ktb
-            case .internetBankingSCB:
-                bank = .scb
-            case .mobileBankingSCB:
-                bank = .scb
```

Comment:
Shouldn't we delete this? 🤔

Reply:
I removed this one because if you could see the whole function this one is for `__SourceInternetBankingPayment` ka, and there are no other mobile bankings in this source payment switch case.